docs/building_pybindings.txt

```diff
@@ -0,0 +1,99 @@
+Some of the project's librariers (traffic, search, kml and others) come
+with python bindings, i.e. have some of their functions exposed to be
+callable from the python programming language. The version of python
+is not enforced, so some libraries may be using python 2 and some may
+be using python 3.
+
+This document describes how to build these python bindings.
+
+
+Installing Boost.Python
+-----------------------
+
+The MAPS.ME project comes with its own version of Boost (https://www.boost.org/)
+in a submodule. While most of the project uses boost in the header-only mode,
+you will need to build the Boost.Python library to link against if you need
+python bindings.
+
+Once you've run
+
+  $ git submodule update --init --recursive
+
+and
+
+  $ ./configure.sh
+
+from omim's root directory, you'll have the Boost headers installed but the
+python library will not be built by default, so we'll need to build it
+manually. (The less clean alternative is to
+link against the version of library installed in your system in case you have
+a system installation of Boost. This is not recommended.)
+
+The official instruction for Boost 1.68 is here: https://www.boost.org/doc/libs/1_68_0/libs/python/doc/html/building/configuring_boost_build.html.
+
+Here is a recap.
+
+Write the paths to your python binary, headers and libraries in a .jam-file.
+We will edit the auto-generated file omim/3party/boost/project-config.jam
+which omim's configure.sh should have generated by now.
+Unfortunately, Boost's bootstrap.sh (called from omim's configures.sh) that generates
+this file is not sufficiently versatile to allow us provide the necessary paths
+at the ./configure.sh stage.
+
+Example for a macOS installation:
+
+    $ pwd
+
+    /omim/3party/boost
+
+    $ cat >>user-config.jam <<EOF
+
+using python
+  : 3.7
+  : /usr/local/opt/python3/Frameworks/Python.framework/Versions/3.7/bin/python3
+  : /usr/local/opt/python3/Frameworks/Python.framework/Versions/3.7/Headers
+  : /usr/local/opt/python3/Frameworks/Python.framework/Versions/3.7/lib
+  ;
+EOF
+
+    $ ./b2 --with-python
+
+The syntax is
+    using python : <version> : <python binary> : <path to python headers> : <path to python libraries> ;
+
+Newlines are allowed, semicolon is required.
+If there are several versions of Boost.Python that you want to build,
+they can all be added to the same .jam-file with this "using python" directive.
+
+To remove the python libraries, execute the script that installed them with the
+additional "--clean" flag:
+
+    $ ./b2 --with-python --clean
+
+
+
+Building pybindings
+-------------------
+
+Now you're ready to build omim's CMake targets that link against Boost.Python:
+
+    $ pwd
+
+    /omim
```

Comment:
`/Users/username/omim`

Reply:
No. The current path is as clear as it could be.

Reply:
okey, but you create `../build_dir` next line. That why I wrote this comment

Reply:
../build_dir = /buid_dir

Reply:
ping about this comment

Reply:
What's wrong with creating ```../build_dir```?
```..``` is ```/```
so
```../build_dir``` is ```/build_dir```